docs/toc.md

```diff
@@ -254,7 +254,7 @@
 ## [Tour of C#](csharp/tour-of-csharp/)
 <!-- The "What's New" section is short, and one level
     deep, so leave it in the main TOC -->
-## [What's new in C#](csharp/whats-new/index.md)
+## [What's new in C#]
```

Comment:
I think that the brackets can be eliminated. They appear in the TOC, the only case in which brackets are used.